static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -297,7 +297,10 @@ const show = () =>
 				const cached = (
 					<CacheProvider value={emotionCache}>
 						<Component
-							componentName={message.extras.componentName}
+							componentName={
+								message.extras.ophanComponentId ??
```

Comment:
I don't think this is what we want. This passes the `ophanComponentId` as the component name to the high level entry point `<Component />`. There won't be a component with the name of the ophan component ID so this will result in us not rendering anything when there is a `ophanComponentId` present.

I think in order to be equivalent with guardian/dotcom-rendering#3776 we want to be passing the `ophanComponentId` if there is one to `submitViewEvent` which is a little further down in this file, falling back on `componentName` if there isn't.

Reply:
You're right, I've pushed a change to fix this.